Assets/GothicVR/Scripts/World/DijkstraPathFinder.cs

```diff
@@ -0,0 +1,328 @@
+using UnityEngine;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using GVR.Extensions;
+using PxCs.Interface;
+using GVR.Phoenix.Data;
+using System.Collections;
+using GVR.Vob.WayNet;
+using System.Linq;
+using System;
+using System.IO;
+
+namespace GVR
+{
+
+    public class DijkstraPathFinder : MonoBehaviour
```

Comment:
I would put this to Assets/GothicVR/Scripts/Manager/WaynetHelper.cs tbh.

I also didn't check for the code itself. I believe in you. :-D

It would be awesome, if we could create this path inside the Lab. This way we would have a proper integration test.
I can explain Lab to you these days.

Reply:
In the WaynetHelper.cs class, we also work with static class. This is due to the fact, that we only use MonoBehaviour (GO-Components), if needed.
Based on our knowledge: The more we put into a scene, the more merge conflicts we create. So the helper/creator, ... are mostly static classes if no Unity magic like Update() etc. is needed.

Reply:
Updated.